### PR TITLE
Fix/minor changes

### PR DESCRIPTION
**Pull request by toto-b**

### CODE REVIEW

File: WagerMate/Components/Pages/Bet-overview.razor [hunk 1]
```diff
@@ -319,7 +329,7 @@
             FilterBets();
         }
     }
-    private bool _displayClosedBets { get; set; } = false;
+    private bool _displayClosedBets { get; set; } = true;
```
Comment (itsAnian) [inline WagerMate/Components/Pages/Bet-overview.razor:332]: Why true instead of false now?

File: WagerMate/Components/Pages/Bet-overview.razor [hunk 2]
```diff
@@ -332,7 +342,6 @@
     private User? _loggedInUser = new();
     private FormBetDetails _formBetDetails = new();
     private string? ErrorMessage { get; set; }
-    private int _totalAmount { get; set; }
```
Comment (itsAnian) [inline WagerMate/Components/Pages/Bet-overview.razor:335]: Where is totalAmount now?

Comment (toto-b) [reply WagerMate/Components/Pages/Bet-overview.razor:335]: No longer in use due to dynamically loading the amount of bets

File: WagerMate/Components/Pages/Bet-overview.razor (hunk 1 above)
Comment (toto-b) [reply WagerMate/Components/Pages/Bet-overview.razor:332]: We decided to display all bet-states when initially loading

**Review by itsAnian — APPROVED** [review]
lgtm

**Pull request merged**